TESTS/configs/baremetal.json

```diff
@@ -0,0 +1,40 @@
+{ 
```

Comment:
Add "rtos-api" here and there is no need to drop sleep calls in PR #11825

Reply:
Yes! That will help enable many more tests.